src/main/java/com/github/dockerjava/core/command/CreateContainerCmdImpl.java

```diff
@@ -590,6 +596,13 @@ public CreateContainerCmd withCpuPeriod(Integer cpuPeriod) {
     }
 
     @Override
+    public CreateContainerCmd withCpuQuota(Integer cpuQuota) {
+        checkNotNull(cpuQuota, "cpuQuota was not specified");
+        hostConfig.withCpuQuota(cpuQuota);
```

Comment:
I just made withCpuQuota behave exactly like withCpuPeriod.  Are you thinking that withCpuQuota should be different from withCpuPeriod? If so, why is withCpuQuota different from withCpuPeriod?